test/resolver.spec.ts

```diff
@@ -644,7 +644,7 @@ describe('resolver', () => {
         }
       }
 
-      @autoinject()
+      @autoinject
```

Comment:
Maybe copy paste these tests to verify both syntaxes? I'd like to be sure the other syntax also still works